notification/alertstatus.go

```diff
@@ -5,6 +5,14 @@ type AlertStatus struct {
 	CallbackID string
 	AlertID    int
 	LogEntry   string
+
+	// Summary of the alert that this status is in regards to.
+	Summary string
+	// Details of the alert that this status is in regards to.
+	Details string
+
+	// OriginalStatus is the status of the first Alert notification to this Dest for this AlertID.
+	OriginalStatus *SendResult
```

Comment:
Make this a value instead of a pointer since it's always required & guaranteed for status. Eliminates the need for unnecessary nil checks elsewhere.

** will need to update a few other places for the type change (compile errors)

```suggestion
	OriginalStatus SendResult
```